test_scripts/Polices/External_UCS/016_ATF_N_TC_PTU_with_external_consent_status_groups_struct.lua

```diff
@@ -0,0 +1,127 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU with "external_consent_status_groups" struct
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate with "external_consent_status_groups:
+-- [<functional_grouping>: <Boolean>]” -> of "device_data" -> "<device identifier>"
+-- -> "user_consent_records" -> "<app id>" section
+-- SDL must:
+-- a. consider this PTU as invalid
+-- b. do not merge this invalid PTU to LocalPT.
+--
+-- Preconditions:
+-- 1. Modify PreloadedPT by adding 'disallowed_by_external_consent_entities_on' section
+--
+-- Steps:
+-- 1. Register app
+-- 2. Send SDL.OnAppPermissionConsent (change ON -> OFF)
```

Comment:
@dboltovskyi, the idea of script is to verify that PTU will fail in case disallowed_by_external_consent_entities_on is observed in ptu file. I think that this step is redundant.